docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -187,6 +187,11 @@ To enable consistent hashing for a backend:
 
 `nginx.ingress.kubernetes.io/upstream-hash-by`: the nginx variable, text value or any combination thereof to use for consistent hashing. For example `nginx.ingress.kubernetes.io/upstream-hash-by: "$request_uri"` to consistently hash upstream requests by the current request URI.
 
+"subset" hashing can be enabled using 
+`nginx.ingress.kubernetes.io/upstream-hash-type`: subset. This map requests to
```

Comment:
Is there going to be any other type? If not can we call this something like `nginx.ingress.kubernetes.io/upstream-hash-subset` and make it boolean defaulting to `false`?